METADATOS/Meta 6.py

```diff
@@ -0,0 +1,60 @@
+#
+# @video_converter.py Copyright (c) 2022 Jalasoft
+# 2643 Av Melchor Perez de Olguin , Colquiri Sud, Cochabamba, Bolivia.
+# add direccion de jala la paz>
+# All rights reserved
+#
+# This software is the confidential and proprietary information of
+# Jalasoft , Confidential Information "). You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft
+
+
+import subprocess
+import os
+
+##CREAMOS NUESTRA CLASE
+class Metadata:
+
+### DEFINIMOS LAS ENTRADAS DE LA CLASE
+    def __init__(self,meti):
+        self.path = "C:/IVAN/CODING/Metadatos/filein"
```

Comment:
Can you change this paths lo local paths instead of absolute paths please

Reply:
is it okay that the comments are in spanish? review please